synapse/federation/federation_server.py

```diff
@@ -861,7 +862,7 @@ def __init__(self, hs: "HomeServer"):
         self.query_handlers = {}  # type: Dict[str, Callable[[dict], Awaitable[None]]]
 
         # Map from type to instance name that we should route EDU handling to.
```

Comment:
could you update this comment?  what do multiple instance names mean?

Reply:
done